hms-app/src/actions/IdeaActions.tsx

```diff
@@ -64,3 +64,37 @@ export const createIdea = (
     })
     .catch((err) => console.log(err))
 }
+
+export const editIdea = (
+  ideaID: string,
+  idea: {
+    ownerId: string
+    hackathonId: string
+    title: string
+    description: string
+    problem: string
+    goal: string
+  },
+  skills: string[],
+  categories: string[]
+) => {
+  return fetch(`${core_url}/idea/${ideaID}`, {
+    method: 'PUT',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      title: idea.title,
+      description: idea.description,
+      problem: idea.problem,
+      goal: idea.goal,
+      requiredSkills: skills,
+      categoryId: categories.toString(),
```

Comment:
this will do something weird, as it is still a single category in the backend

Reply:
lets assume there is only 1 category in the frontend for now

Reply:
yeah, but this will convert it to a string like `["aabeb-12234nb..."]` with the brackets around it and the backend won't (or shouldn't) like it